packages/kolibri-common/components/SearchFiltersPanel/AccordionSelectGroup.vue

```diff
@@ -412,31 +408,32 @@
     margin-bottom: 1em;
   }
 
+  .category-button {
+    position: relative;
+    display: flex;
+    align-items: center;
+    width: 100%;
+    padding: 0.25em 0.5em;
+    font-weight: normal;
+    text-align: left;
+    text-transform: none;
+  }
+
   .category-icon {
-    position: absolute;
-    top: 50%;
-    left: 0.5em;
-    width: 32px;
-    height: 32px;
-    transform: translateY(-50%);
+    order: 1;
+    width: 24px;
+    height: 24px;
+    margin-right: 1em;
   }
 
   .category-icon-after {
     position: absolute;
-    top: 50%;
+    top: 0.75em;
     right: 0.5em;
-    transform: translateY(-50%);
-  }
-
-  .category-button {
-    // Ensure the child KIcons' absolute positioning anchors to this button
-    position: relative;
-    width: 100%;
-    padding: 0.25em 0.5em 0.25em 3.5em;
-    font-weight: normal;
-    text-align: left;
-    // KButton text formatting overrides
-    text-transform: unset;
+    order: 3;
```

Comment:
This order style neither. At the end, an absolute-positioned item within a flex container is ignored by flex, so any flex property we apply to it will be ignored. It wont have any effect if we remove it or keep it here.

Reply:
I have removed them, just to reduce confusion for our future selves. Thanks - Alex. My flexbox knowledge is growing but I still have a ways to go 😂 Flexbox froggy is on my to do list